src/main/java/seedu/duke/Parser.java

```diff
@@ -55,19 +55,19 @@ public Commands parse(String input) {
      */
     public int mark(String input) throws DukeException {
         if (input.length() == 4) {
-            throw new DukeException(Ui.i5 + "☹ OOPS!!! Please specify a valid task number to be marked.");
+            throw new DukeException(Ui.I5 + "☹ OOPS!!! Please specify a valid task number to be marked.");
         }
         if (input.charAt(4) != ' ') {
-            throw new DukeException(Ui.i5 + "☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+            throw new DukeException(Ui.I5 + "☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
         }
         if (input.length() == 5) {
-            throw new DukeException(Ui.i5 + "☹ OOPS!!! Please specify a valid task number to be marked.");
+            throw new DukeException(Ui.I5 + "☹ OOPS!!! Please specify a valid task number to be marked.");
         }
 
         try {
             return Integer.parseInt(input.substring(5)) - 1;
         } catch (NumberFormatException e) {
-            throw new DukeException(Ui.i5 + "☹ OOPS!!! Please specify a valid task number to be marked.");
+            throw new DukeException(Ui.I5 + "☹ OOPS!!! Please specify a valid task number to be marked.");
         }
```

Comment:
Any reason why you added the _try-catch block_ in the code even though you have _if statements_ that have handled several exceptions?